web/client/plugins/Annotations.jsx

```diff
@@ -194,7 +195,10 @@ const conditionalToggle = on.bind(null, toggleControl('annotations', null), (sta
   * If an odd number of values is inserted then they are added again to reach an even number of values
   * for more information see [this page](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray)
   * @prop {string} symbolsPath the relative path to the symbols folder where symbols.json and SVGs are located (starting from the index.html folder, i.e. the root)
-  * @prop {string} defaultShape the default symbol used when switching to the symbol styler from marker styler
+  * @prop {string} defaultShape the default symbol used when switching for the symbol styler
+  * @prop {string} defaultShapeStrokeColor default symbol stroke color
+  * @prop {string} defaultShapeFillColor default symbol fill color
+  * @prop {string} defaultShapeSize default symbol shape size in px
```

Comment:
also here is is missing defaultPointType 
```suggestion
  * @prop {string} defaultShapeSize default symbol shape size in px
  * @prop {string} defaultPointType default point type, can be "symbol" or "marker", default is "marker"

```